src/components/CategoryCardList.tsx

```
@@ -0,0 +1,16 @@
+import { useSelector } from 'react-redux'
+import { RootState } from '../redux/store'
+import { CategoryCard } from './CategoryCard'
+import { Category } from '../types'
+
+export const CategoryCardList = () => {
+  const categories = useSelector((state: RootState) => state.categories)
+
+  return (
+    <div className='flex space-x-4 p-4'>
+      {categories.map((category: Category) => (
+        <CategoryCard category={category} />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)